awscli/topics/s3-config.rst

```diff
@@ -15,6 +15,11 @@ are provided in the case where you need to modify one of these values, either
 for performance reasons or to account for the specific environment where these
 ``aws s3`` commands are being run.
 
+.. warning::
+   S3 commands have a parameter ``--endpoint-url`` that overrides the command's
+   default URL with the given URL. Avoid using the S3 Global Endpoint
```

Comment:
I think we could be a bit more general here and just say you should avoid configuring the endpoint URL at all when using the AWS S3 service? Generally endpoint URL is only going to be needed for 3P endpoints when using S3.